x/emissions/module/rewards/rewards_internal.go

```diff
@@ -310,35 +282,27 @@ func GetStakeWeightedLossMatrix(
 			}
 		}
 
-		logSum := alloraMath.ZeroDec()
+		sum := alloraMath.ZeroDec()
 		for i, losses := range reputersReportedLosses {
 			// Skip if loss is NaN
 			if losses[j].IsNaN() {
 				continue
 			}
 
-			logLosses, err := alloraMath.Log10(losses[j])
-			if err != nil {
-				return nil, nil, err
-			}
-			logLossesTimesStake, err := logLosses.Mul(reputersAdjustedStakes[i])
+			lossesTimesStake, err := losses[j].Mul(reputersAdjustedStakes[i])
 			if err != nil {
 				return nil, nil, err
 			}
-			logLossesTimesStakeOverTotalStake, err := logLossesTimesStake.Quo(totalStakeToConsider)
+			lossesTimesStakeOverTotalStake, err := lossesTimesStake.Quo(totalStakeToConsider)
```

Comment:
meanwhile, this is an instance of `.Quo()` that definitely should be kept as-is (to be clear, I'm saying this code, as you coded it, is exactly right)